### ISSUE TITLE
Save_Anywhere: unhandled exception when saving in the community centre

### ISSUE THREAD

**Issue by Jarvie8176** [janavarro95/Stardew_Valley_Mods#11]
## How to reproduce

save in the community centre with a Junimo sprite in there.

## Details

the mod creates a custom npc (at least from the log at `Save_Anywhere_V2/NPC_Utilities.cs#L47`) then there's an unhandled exception printed in SMAPI's console

```
An error occured in the overridden update loop: System.InvalidOperationException
```

when generating XML document

The game is then stuck at saving. Next time loading the save, player location and time were set correctly but everything else (inventory, bundles) were still in the states of the last successful save.

**Comment by janavarro95** [comment]
Ohh right. I forgot that I originally had a fix for this but the old library I was using got outdated.

I'll have to fix this in the next update which should hopefully be in about a weeks time since I'm away on travels at the moment.

**Comment by Pathoschild** [comment]
Fixed in #22.
